### PR TITLE
Reorg to support adoptium

### PR DESCRIPTION
**Pull request by johnoliver**
# Checklist

- [ ] You added tests to cover the change
- [ ] `mvn clean install` build and test completes
- [ ] You changed or added to the documentation

### CODE REVIEW

File: adoptopenjdk-frontend-parent/adoptium-frontend-assets/pom.xml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>net.adoptopenjdk.api</groupId>
+        <artifactId>adoptopenjdk-frontend-parent</artifactId>
```
Comment (karianna) [inline adoptopenjdk-frontend-parent/adoptium-frontend-assets/pom.xml:7]: folder says adoptium-frontend-assets but group id is adoptopenjdk-frontend-parents

**Review by karianna — COMMENTED** [review]
This looks mostly OK, I assume the addition of adoptium is simply for the forwarding to Adoptium API for eclipse Temurin binaries?

**Comment by karianna** [comment]
@johnoliver sorry merged another PR so this needs a rebase.

File: adoptopenjdk-frontend-parent/adoptium-frontend-assets/pom.xml (hunk 1 above)
Comment (johnoliver) [reply adoptopenjdk-frontend-parent/adoptium-frontend-assets/pom.xml:7]: that is the parent id

**Pull request merged**